### ISSUE TITLE
⚡ Feature | Register username page

### ISSUE THREAD

**Issue by pradel** [sigle/sigle#366]
## Description

After the Hiro wallet integration we need to create a new page for the registration of new usernames:
- the login layout can probably be shared between the login page and this one
- path should be `/register-username`
- accessing the page with a username should redirect you to the dashboard
- as the API to register a domain is slow the button should have a loading indicator (see with Quentin for the designs)
- the link is https://btc.us/ to register a name
- the interrogation mark should open a new tab to https://docs.sigle.io/ (we will change it once the documentation is ready)

This task is about the layout only, the logic to register the username will be added in another pull request.

![Screenshot 2022-03-03 at 10 47 03](https://user-images.githubusercontent.com/5749437/156539449-65475687-5476-4e5d-b976-7d4d9783e612.png)

**Comment by gregogun** [comment]
Regarding the shared layout between this and login page, just to be clear, dyt it would be a good idea create a shared layout module for similar components, or is it fine to copy and paste over (doing this currently) ?

**Comment by gregogun** [comment]
Also, is the `isExperimentalHiroWalletEnabled` flag required for this page, as on the login page?

**Comment by pradel** [comment]
Regarding the shared layout my initial idea was to create some kind of LoginLayout that can be shared by both components. The layout would contain everything and accept a children prop that would be injected in the left column.

> Also, is the isExperimentalHiroWalletEnabled flag required for this page, as on the login page?
Not on this one as it's a different path, all good :)
